go.mod

```diff
@@ -20,6 +20,10 @@ require (
 	github.com/andres-erbsen/clock v0.0.0-20160526145045-9e14626cd129
 	github.com/armon/go-metrics v0.3.2
 	github.com/aws/aws-sdk-go v1.28.9
```

Comment:
I iknow it is not related to this PR, but, is there a plan about moving our plugins to use V2?

Reply:
I took the liberty of creating one https://github.com/spiffe/spire/issues/2145